style.css

```diff
@@ -601,3 +794,477 @@ button:hover {
   left: 280px;
   top: 5110px;
 }
+
+@media screen and (min-width: 768px) {
+  .container {
+    width: 1440px;
+    display: flex;
+    flex-direction: column;
+    border: solid;
+    margin: 0;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .header_container {
+    display: flex;
+  }
+
+  .nav_ul {
+    display: flex;
+    align-items: center;
+    margin-right: 100px;
+    justify-content: space-around;
+    width: 259px;
+    font-family: 'Poppins', sans-serif;
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+    line-height: 20px;
+    gap: 8px;
+  }
+
+  .menuim {
+    display: none;
+  }
+
+  .input {
+    padding-left: 143px;
+  }
+
+  .arc {
+    width: 224px;
+    height: 224px;
+    left: 1180px;
+    top: 191px;
+  }
+
+  .bar {
+    width: 56.46px;
+    height: 70.73px;
+    left: 693.18px;
+    top: 260.59px;
+  }
+
+  .c {
+    width: 80.5px;
+    height: 40.25px;
+    left: 139px;
+    top: 260.47px;
+  }
+
+  .circle {
+    width: 47px;
+    height: 47px;
+    left: 800px;
+    top: 850.1px;
+  }
+
+  .info_box {
+    display: flex;
+    align-items: center;
+    font-size: 48px;
+    width: 960px;
+    height: 122px;
+    line-height: 60px;
+    margin-left: 260px;
+    margin-right: 260px;
+    font-weight: 700;
+    margin-top: 370px;
+    margin-bottom: 12px;
+    letter-spacing: 0.37px;
+  }
+
+  .body_section {
+    width: 920px;
+    height: 98px;
+    line-height: 28px;
+    font-size: 20px;
+    font-weight: 400;
+    margin-left: 260px;
+    margin-right: 260px;
+  }
+
+  .contact_section {
+    margin-left: 260px;
+  }
+
+  .sec_icn {
+    margin-left: 210px;
+  }
+
+  .section_works_container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 1165px;
+    background: #fff;
+    margin-left: 142px;
+    height: 496px;
+  }
+
+  .sec_in {
+    position: absolute;
+    left: 118px;
+    top: 950px;
+    width: 544px;
+    height: 448px;
+  }
+
+  .tonic {
+    position: absolute;
+    margin-left: 756px;
+    width: 496px;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .sec_h {
+    position: absolute;
+    margin-left: 756px;
+    top: 1030px;
+    width: 297px;
+  }
+
+  .p {
+    display: flex;
+    font-size: 16px;
+    height: 50px;
+    width: 496px;
+    margin-left: 756px;
+    margin-right: 188px;
+    margin-top: 170px;
+  }
+
+  .push {
+    padding-left: 0;
+    width: 203px;
+    margin-left: 756px;
+  }
+
+  button {
+    width: 149px;
+    margin-left: 756px;
+    margin-top: 40px;
+  }
+
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly use the correct padding, and add the correct color and background to all the buttons to indicate the interaction of hover, active, enabled, and disabled to match the Figma design as you've used another background color and the other states of the buttons haven't been implemented yet . see the image below:

![buton](https://user-images.githubusercontent.com/59397159/172934090-cfdc0d32-80cc-4261-9362-197ad6da053c.png)